backend/main.py

```diff
@@ -123,32 +123,48 @@ async def websocket_endpoint(websocket: WebSocket):
             elif message["type"] == "bid":
                 if not game.ongoing:
```

Comment:
So game ongoing means we're either in waiting or playing state? I'm not sure about the flow here, or naming of the states.